File: ControlRemote/Web/Dto/ActionPointAtTimeModel.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Web.Dto
{
    public class ActionPointAtTimeModel
    {
        public DateTime HourTimeAction { get; set; }
        public byte FlagImg { get; set; }
        public bool EnableAction { get; set; }
    }
}

File: ControlRemote/Web/ClientApp/src/app/Dto/ActionPointAtTimeModel.ts
import { Byte } from "@angular/compiler/src/util";

export class ActionPointAtTimeModel {
    hourTimeAction: Date;
    flagImg: Byte;
    enableAction: boolean;

    public constructor(_hourTimeAction: Date, _flagImg: Byte, _enableAction: boolean) {
        this.hourTimeAction = _hourTimeAction;
        this.flagImg = _flagImg;
        this.enableAction = _enableAction;
    }
}